Description: Use ${root} in the generated drivemap menuentry. Fixes
 chain-loading.
Upstream: Backported from svn://svn.sv.gnu.org/grub/trunk/grub2 r2333.

diff -Nur -x '*.orig' -x '*~' grub2-1.96+20090611/util/grub.d/30_os-prober.in grub2-1.96+20090611.new/util/grub.d/30_os-prober.in
--- grub2-1.96+20090611/util/grub.d/30_os-prober.in	2009-06-11 20:00:49.000000000 +0100
+++ grub2-1.96+20090611.new/util/grub.d/30_os-prober.in	2009-06-17 12:02:57.000000000 +0100
@@ -55,7 +55,7 @@
       prepare_grub_to_access_device ${DEVICE} | sed -e "s/^/\t/"
 
       cat << EOF
-	drivemap -s (hd0) \$root
+	drivemap -s (hd0) \${root}
 	chainloader +1
 }
 EOF
